presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java

```diff
@@ -157,6 +157,18 @@ public void stop()
         queryPurger.shutdownNow();
     }
 
+    /**
+     * Creates new queries on Presto.
+     * <p>
+     * Receives instruction requests that create new queries on Presto.
+     *
+     * @param statement a sql command.
+     * @param servletRequest an object that provides request information for HTTP servlets.
+     * @param httpHeaders an object responsible for the http request headers.
+     * @param uriInfo an object responsible for taking information from the uri.
```

Comment:
an object containing all extra information that should be added to the uri.
[Confirm with Caio]